src/views/PurchaseOrder.vue

```diff
@@ -121,19 +123,26 @@ export default defineComponent({
       mapFields() {
         this.orderItemsList = this.content.map(item => {
```

Comment:
We can move this array creation logic just before dispatching `updatedOrderList` action, because it might be a case that some fields are not selected.

Reply:
Made the requested change